go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	golang.org/x/net v0.29.0 // indirect
```

Comment:
I think you shouldn't have any change in go.mod, go.sum. You can possibly revert these files?